repo2docker/buildpacks/base.py

```diff
@@ -117,6 +117,8 @@
 # Copy and chown stuff. This doubles the size of the repo, because
 # you can't actually copy as USER, only as root! Thanks, Docker!
 USER root
+# Explicitly create $HOME, in case $REPO_DIR is explicitly set to be somewhere
+RUN mkdir "${HOME}" && chown ${NB_USER}:${NB_USER} "${HOME}"
```

Comment:
```suggestion
RUN mkdir -p "${HOME}" && chown ${NB_USER}:${NB_USER} "${HOME}"
```

This fails currently with

```
mkdir: cannot create directory ‘/home/travis’: File exists
```

The -p fixes that. Also, shouldn't adduser already be creating this?